### PR TITLE
Add Koding Readme

### PR DESCRIPTION
**Pull request by sibblegp**

### CODE REVIEW

**Comment by ryadav88** [comment]
thanks @sibblegp 

@mhausenblas for you to review and merge

File: 1.8/koding/README.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
+# How to use Koding on DC/OS
+
+[Koding](https://wwww.koding.com/) is cloud development platform.
```
Comment (mhausenblas) [inline 1.8/koding/README.md]: Can you please elaborate on this? What for?

File: 1.8/koding/README.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+- [Prerequisites](#prerequisites)
+- [Install Marathon-LB](#install-marathon-lb)
+- [Install Koding](#install-koding)
+- [Access Koding](#access-koding)
+- [Use Koding in production](#use-koding-in-production)
+
+## Prerequisites
+
+- A running DC/OS 1.8 cluster with at least 1 [private agent](https://dcos.io/docs/1.8/overview/concepts/#public) node with 2 CPUs and 1 GB of RAM available.
```
Comment (mhausenblas) [inline 1.8/koding/README.md]: This differs from the statement below 'requires 2 CPUs and 8GB of RAM'. Suggest that you only state the requirements here and remove below.

File: 1.8/koding/README.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+You will then be able to allocate CPUs and memory to Koding. To run at its best, Koding requires 2 CPUs and 8GB of RAM, although it can run with a minimum of 1GB of RAM.
+
+Next, you will need to pick which node to run Koding on. This is to preserve data across restarts of the Koding service. Select a node with enough available resources (as you just defined) from your available private agent nodes.
+
+After you select a node, you will be asked for a hostname. This hostname is used within the Koding application to define where Koding lives. Individual teams will live at subdomains of this hostname. It mostly exists to be displayed inside the application, but should be where you intend Koding to be accessed from. You may want to make a CNAME record of this host and a wildcard record for any subdomains to point to the FQDN you set up in the next step.
+
+Lastly, to make Koding accessible to the world, we need to use the fully qualified domain name of the public agent node running marathon-lb or the load balancer pointing to your public agent nodes (for instance "DCOS-PublicSlaveLo-1CHPNZSAH97SJ-604928754.eu-west-1.elb.amazonaws.com").
```
Comment (mhausenblas) [inline 1.8/koding/README.md]: Node names are written using code, for example `DCOS-PublicSlaveLo-1CHPNZSAH97SJ-604928754.eu-west-1.elb.amazonaws.com`

File: 1.8/koding/README.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+
+After all options are configured, press Review and Install and the Install to set up Koding.  Please note that it may take up to 10 minutes for the Koding service to become available.
+
+Last but not least, to check if all required services (Marathon-LB and Koding itself) are running, use the DC/OS UI where in the `Services` tab you should see both Marathon-LB and Koding listed with their statuses as "Running".
```
Comment (mhausenblas) [inline 1.8/koding/README.md]: Add screen shot here please.

File: 1.8/koding/README.md [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+
+## Access Koding
+
+Once you've installed the necessary services as outlined above, navigate to the domain name that you set as your virtual host in your browser. You should now see the Koding login page.
```
Comment (mhausenblas) [inline 1.8/koding/README.md]: Add screen shot here please.

**Comment by mhausenblas** [comment]
@ryadav88 @sibblegp any chance you address above comments so that we can publish the example?

File: 1.8/koding/README.md (hunk 3 above)
Comment (sibblegp) [inline 1.8/koding/README.md]: I don't quite follow what you are trying to say right here.  Isn't that what I'm suggesting in this line?  I've used that exact line as the VHOST parameter and it's worked before.

File: 1.8/koding/README.md (hunk 4 above)
Comment (sibblegp) [inline 1.8/koding/README.md]: Added

**Comment by sibblegp** [comment]
@mhausenblas Changes made except for the node part.  I'm not sure what you are asking for.

File: 1.8/koding/README.md (hunk 3 above)
Comment (mhausenblas) [inline 1.8/koding/README.md]: Sorry, it's simply a matter of formatting: `EXAMPLE` vs "EXAMPLE" (so the Markdown syntax for inline code which are back ticks @sibblegp … makes sense?

**Comment by sibblegp** [comment]
@mhausenblas Change to node code made.

**Comment by mhausenblas** [comment]
Thanks again @sibblegp and @ryadav88 … will merge as soon as https://github.com/mesosphere/universe/pull/796 is merged in and the package is available in the Universe.

**Comment by ryadav88** [comment]
@sibblegp can you add a note to the example here about removing `haproxy-map` from marathon-lb

**Comment by sibblegp** [comment]
@mhausenblas haproxy instructions added.

**Comment by mhausenblas** [comment]
Thank you so much for this @sibblegp and also kudos to @ryadav88 for shepherding it!

**Pull request merged**